core/src/main/java/de/danoeh/antennapod/core/service/download/DownloadRequest.java

```diff
@@ -38,50 +39,76 @@ public DownloadRequest(@NonNull String destination,
                            String username,
                            String password,
                            boolean deleteOnFailure,
-                           Bundle arguments) {
+                           Bundle arguments,
+                           boolean initiatedByUser) {
+        this(destination,
+             source,
+             title,
+             feedfileId,
+             feedfileType,
+             null,
+             deleteOnFailure,
+             username,
+             password,
+             false,
+             arguments,
+             initiatedByUser);
```

Comment:
To be honest, I think this is harder to read than before. This nearly fills the screen of my laptop vertically and is basically empty horizontally. Please break the lines, so that one line has more than one parameter.

Reply:
Changed.